src/scss/components/_ratings.scss

```diff
@@ -13,6 +14,11 @@ $star-spacing: $star-count;
     width: $star-size-small * $star-count + $star-spacing;
 }
 
+.c-rating--extraSmall {
+    height: $star-size-extraSmall;
+    width: $star-size-extraSmall * $star-count + $star-spacing;
```

Comment:
Should this calculation be in brackets?

($star-size-extraSmall * $star-count) + $star-spacing

Reply:
for the calculation, no. for readability, yeah! - Have updated them all 👍